src/main/java/com/buuz135/industrial/tile/world/BlockDestroyerTile.java

```diff
@@ -72,6 +83,10 @@ public float work() {
                     for (ItemStack stack : drops) {
                         ItemHandlerHelper.insertItem(outItems, stack, false);
                     }
+                    if(tile instanceof TileEntityShulkerBox) {
```

Comment:
maybe even do this more generically for all tiles that are IInventory or have the inventory capability thingy...

Reply:
The problem is only with the shulkerbox, can you explain me the advantage to check for all IIventory tile please ? I'm new on the minecraft modding thing

Reply:
I guess shulker box is a special case for whatever that machines does... but it's special because it has an inventory... probably :)

there are many many many things that have inventory... and thus might have the same issue.

tbh, I have no idea what that machine does or what the original issue was... :D :D